v3/.vscode/settings.json

```diff
@@ -22,5 +22,5 @@
   ],
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
-  "files.trimTrailingWhitespace": true
+  "files.trimTrailingWhitespace": false
```

Comment:
I have some workflow quirks where I like to leave a trailing whitespace for a bit. I'll undo the change, though down the line I might try adding Prettier so that we can enforce consistent styling without requiring certain editor settings.